mobile-widgets/qml/Settings.qml

```diff
@@ -45,7 +45,7 @@ Kirigami.ScrollablePage {
 				color: subsurfaceTheme.textColor
 			}
 			Controls.Label {
-				text: prefs.credentialStatus === CloudStatus.CS_NOCLOUD ? qsTr("Not applicable") : prefs.cloudUserName
+				text: prefs.credentialStatus === CloudStatus.CS_NOCLOUD ? qsTr("Not applicable") : PrefCloudStorage.cloud_storage_email
```

Comment:
overall I still find the way we access these values from QML highly inconsistent and confusing.
Here we are now using `prefs` and `PrefCloudStorage` on the same line...

Reply:
Sadly enough prefs.credentialstatus is a calculated value in mobile-widgets/prefs.cpp, otherwise I would have replace that. I am working on a bigger cleanup of the login procedure to highly simplify it (but keep exact the same functionality), and this commit is basically part of that.

I agree that the QML - C++ interface not is the best in the world, I am slowly trying to make it better, by e.g. eliminating prefs (which are a leftover from the early days of qPref and thus in reality a parallel implementation to qPref just to add confusion.

I do believe with every patch the interface get clearer and more precisely defined.